docker-compose.yml

```diff
@@ -8,14 +8,15 @@ services:
     restart: always
 
   gitcollector:
-    image: srcd/gitcollector:v0.0.1-beta.5
+    image: srcd/gitcollector:v0.0.1-beta.6
     # wait for db
     command: ['/bin/sh', '-c', 'sleep 10s && gitcollector download']
     environment:
       GITHUB_ORGANIZATIONS: ${GITHUB_ORGANIZATIONS:-}
       GITHUB_TOKEN: ${GITHUB_TOKEN:-}
       # use main db
       GITCOLLECTOR_METRICS_DB_URI: postgresql://superset:superset@postgres:5432/superset?sslmode=disable
+      GITCOLLECTOR_HALF_CPU: "true"
```

Comment:
So we don't want to expose this to the user through an env var as it's always better to have it set to `"true"`?

Reply:
user can't change env variables (normally). Our cli generates `.env` without taking into account any configuration from the user's environment. If it's an advanced user they can change `docker-compose.yml` as well.